downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc

```diff
@@ -11,4 +11,6 @@ The following procedures form the user configuration:
 * xref:eda-set-up-new-project[Setting up a new project]
 * xref:eda-set-up-new-decision-environment[Setting up a new decision environment]
 * xref:eda-set-up-token[Setting up a token to authenticate to {PlatformNameShort} Controller]
-* xref:eda-set-up-rulebook-activation[Setting up a rulebook activation]
\ No newline at end of file
+* xref:eda-set-up-rulebook-activation[Setting up a rulebook activation]
+
+NOTE: API documentation for {EDAcontroller} is available at https://[eda-server-host]/api/eda/v1/docs
```

Comment:
Use the admonition formatting for this as noted in the [Red Hat Supplementary Guide, under Structure > Admonitions](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions): 

```suggestion
[NOTE]
====
API documentation for {EDAcontroller} is available at https://[eda-server-host]/api/eda/v1/docs
====
```

Also, is this link designed to take them to this location: https://github.com/ansible/eda-server ? When I check the link to see where it takes me, it lands me in a browser page with search results pointing to a variety of EDA github pages. Just want to be sure it lands where the reporter for the jira intended.